packages/x-components/tests/e2e/common/mocked-responses.spec.ts

```diff
@@ -520,16 +522,22 @@ Given('a results API response for a misspelled word', () => {
   });
 });
 
-// Suggestions
-Given('a suggestions API', () => {
-  cy.intercept(getSuggestionsEndpoint, req => {
+Given('a query suggestions API', () => {
```

Comment:
Restore the comment // QuerySuggestions

Reply:
We decided to remove this kind of comments due to them not providing any useful information. The `given` clause text should be self explanatory in these cases.